pages/about.html

```diff
@@ -0,0 +1,108 @@
+            </ul>
+            <div class="mobile-menu display-none">
+                <div class="close-mobile-menu">
+                    <button type="button" class="close-btn"><i class="fa-solid fa-xmark fa-3x"></i></button>
+                </div>
+                <ul class="mobile-menu-links">
+                    <li class="mobile-menu-link-items"><a href="../index.html">Home</a></li>
+                    <li class="mobile-menu-link-items"><a href="#program">Program</a></li>
+                    <li class="mobile-menu-link-items"><a href="#join">Join</a></li>
+                    <li class="mobile-menu-link-items"><a href="#sponser">Sponser</a></li>
+                    <li class="mobile-menu-link-items"><a href="#news">News</a></li>
+                </ul>
+            </div>
+        </nav>
+        <div class="headline about-headline">    
+            <h1 class="header-title">United Nations<br> Climate Change<br> Conference 2022</h1>
+            <p class="header-msg about-headline">COP 27 will focus on developing greater resilience and reducing vulnerability to climate change. Finance relates to the $100 billion pledge and the importance of showing progress on delivering this.
+                In particular, it will be a key component of helping to strengthen resilience and achieving these goals.
+            </p>
+            <address class="date-with-location">
+                <p class="date about-address">Please contact us for more information about Conference of the Parties (COP27)</p>
+                <a class="location about-address"><strong>secretariat@unfccc.int</strong></a>
+            </address>
+        </div>
+    </header>
+    <section id="program" class="main-program about-program">
+        <div class="about-display">
+            <h2 class="main-program-header">UN Climate Change Conference Logo</h2>
+            <hr class="red-line">
+            <p class="logo-info mobile-logo-info">The logo of the UN Climate Change Conference represents the global action to make our world safe</p>
+            <p class="logo-info desktop-logo-info">
+                The logo of the UN Climate Change Conference represents the global action to make our world safe.
+                The United Nations Climate Change Conference are yearly conferences held in the framework of the UNFCCC.
+                They serve as the formal meeting of the UNFCCC Parties (Conferences of the Parties) (COP) to assess progress in dealing with climate change, and beginning in the mid-1990s,
+                to negotiate the Kyoto Protocol to establish legally binding obligations for developed countries to reduce their greenhouse gas emissions.
+                This is the symbol that represents that.
+            </p>
+            <img src="../images/logo.png" alt="UN Climate Change Logo" class="about-logo">
+        </div>
+        <div class="logo-hr">
+            <hr>
+        </div>
+    </section>
```

Comment:
- There are some issues with the display on your `about page`(desktop version). Please try to fix these issues.
Your project: 
![about project](https://user-images.githubusercontent.com/73607512/190424622-3f9de5b9-3c7e-43b6-8ddd-65cbbccf3706.PNG)
Design Template: 
![about design template](https://user-images.githubusercontent.com/73607512/190424780-3d300442-2dde-4f33-9516-8d62487b8ad9.PNG)

Reply:
please note that I can't find this issue on my desktop @mckent05